pandas/core/generic.py

```diff
@@ -9055,8 +9057,16 @@ def _doc_parms(cls):
 
 Parameters
 ----------
-axis : int, default 0
-    Select the axis which can be 0 for indices and 1 for columns.
+axis : {None, 0 or 'index', 1 or 'columns'}, default None
```

Comment:
I don't think `default None` is correct? 
The default for *our* method is still 0? It's only when doing `np.all(..)` that we respect the `axis=None` ?